### DISCUSSION TITLE
Change domain used for rancher GUI

### DISCUSSION THREAD

**Discussion by bpopovich44** [rancher/rke2#3862] [General]
I have a rke2 HA cluster consisting of 3 master nodes and 4 worker nodes.  I'm using kube-vip for the load balancer.  I am wanting to change the domain name to a new registered domain on our network, but not sure all the steps.  Can I just change the domain-url in the rancher UI and also the domain names in the tls-san in the /etc/rancher/rke2/config.yaml on all 7 servers and then run a helm upgrade command with new name?

Thank you

**Comment by brandond** [comment]
The tls-san value only controls the certificate on the rke2 apiserver certificate, it is unrelated to anything in Rancher itself. If you want to change the Rancher address and certificate, you'll need to do that in the Rancher configuration. For further questions, I would recommend looking in the Rancher docs, or opening an issue in the rancher/rancher repo.

**Comment by bpopovich44** [comment]
Thanks Brandon.  You've helped me with some other issues and I appreciate it.  My cluster is running great now.  I did find documentation and it seems pretty straight forward, the only question I have is it wants me to run a cluster-agent tool and I can't confirm what that tool exactly does.  I will open an issue on rancher/rancher repo.  

"Once your Rancher deployment is back up and running you need to login to a single controlplane node for each of the downstream clusters and run the cluster-agent-tool. You also need to login to one of the controlplane nodes of your local Rancher cluster and run the cluster-agent-tool. Please see https://github.com/rancherlabs/support-tools/tree/master/cluster-agent-tool"
